t/120-re-find.t

```diff
@@ -891,3 +891,101 @@ not matched!
+            local very_complicated_regex = [[([\d\.]+) ([\w.-]+) ([\w.-]+) (\[.+\]) "([^"\r\n]*|[^"\r\n\[]*\[.+\][^"]+|[^"\r\n]+.[^"]+)" (\d{3}) (\d+|-) ("(?:[^"]|\")+)"? ("(?:[^"]|\")+)"?]]
+            local from, to, err = ngx.re.find(very_long_string, very_complicated_regex, "jo")
+            if from or to then
+                ngx.say("from: ", from)
+                ngx.say("to: ", to)
+            else
+                if err then
+                    ngx.say("error: ", err)
+                    return
+                end
+                ngx.say("not matched!")
+            end
+        }
+    }
+--- request
+    GET /re
+--- response_body
+from: 1
+to: 1563
+--- no_error_log
+[error]
+
+
+
+=== TEST 33: jit_stack_size change disallowed once regex cache is populated
+--- config
+    location /re {
+        content_by_lua_block {
+            local s = "hello, 1234"
+            ngx.re.find(s, "(hello world)|([0-9])", "jo")
+            local status, err = pcall(function() ngx.re.opt("jit_stack_size", 128 * 1024) end)
+            ngx.say(err)
+        }
+    }
+--- request
+    GET /re
+--- response_body
+content_by_lua(nginx.conf:45):4: Changing jit stack size is not allowed when some regexs have already been compiled and cached
+--- no_error_log
+[error]
+
+
+
+=== TEST 34: passing unknown options to ngx.re.opt throws an error
+--- config
+    location /re {
+        content_by_lua_block {
+            local status, err = pcall(function() ngx.re.opt("foo", 123) end)
+            ngx.say(err)
+        }
+    }
+--- request
+    GET /re
+--- response_body
+content_by_lua(nginx.conf:43):2: unrecognized option name
+--- no_error_log
+[error]
```

Comment:
Also need a test case for some regexes have indeed been compiled but *never* cached when `ngx.re.opt("jit_stack_size")` is called.